packages/core/src/components/select/select.tsx

```diff
@@ -346,7 +352,7 @@ export class Select {
     }
 
     if (!value) {
-      this.itemSelectionChange.emit(null);
+      this.itemSelectionChange.emit(undefined);
```

Comment:
```suggestion
      this.itemSelectionChange.emit('');
```

Reply:
The event detail signature is of type `string[]`.